kratos/processes/process.h

```diff
@@ -51,6 +51,8 @@ class Process : public Flags
     ///@name Type Definitions
     ///@{
 
+    using HistoricalVarsMap = std::unordered_map<std::string,std::vector<std::string>>;
```

Comment:
Doc

Reply:
this is just a definition, what documentation you want here?

Reply:
For example: /// The historical variables map. Hash is a variable defined as a string, and it contains a list of variables (vector of strings)
```suggestion
    /// The historical variables map. Hash is a variable defined as a string, and it contains a list of variables (vector of strings)
```

Reply:
you love comments too much... : P